src/DynamoCore/Configuration/Configurations.cs

```diff
@@ -251,9 +251,9 @@ public class Configurations
 
         #region Externally Visible Strings
 
-        internal static readonly string SessionTraceDataXmlTag = "SessionTraceData";
-        internal static readonly string NodeTraceDataXmlTag = "NodeTraceData";
-        internal static readonly string CallsiteTraceDataXmlTag = "CallsiteTraceData";
+        internal static readonly string SessionTraceDataTag = "Bindings";
```

Comment:
why don't you just add an additional const and then use the json one in the json serialization and the xml one in xml deserialization...
I'm also curious did you try adding a serializer specifically for the callSiteTrace data class